From 54bff1522fc863329894d875d54c2fe4cd1b4f3f Mon Sep 17 00:00:00 2001
From: Zachary Henkel <zahen@microsoft.com>
Date: Tue, 4 May 2021 11:19:54 -0400
Subject: [PATCH] Rename a template parameter that conflicted with a common
 macro; NFC

The CALLBACK macro is used extensively in the Windows SDK.
---
 include/clang/Analysis/CFG.h | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/include/clang/Analysis/CFG.h b/include/clang/Analysis/CFG.h
index a0f8c6d21a674..9e32eb8e066a9 100644
--- a/include/clang/Analysis/CFG.h
+++ b/include/clang/Analysis/CFG.h
@@ -1389,13 +1389,12 @@ class CFG {
   // Member templates useful for various batch operations over CFGs.
   //===--------------------------------------------------------------------===//
 
-  template <typename CALLBACK>
-  void VisitBlockStmts(CALLBACK& O) const {
+  template <typename Callback> void VisitBlockStmts(Callback &O) const {
     for (const_iterator I = begin(), E = end(); I != E; ++I)
       for (CFGBlock::const_iterator BI = (*I)->begin(), BE = (*I)->end();
            BI != BE; ++BI) {
         if (Optional<CFGStmt> stmt = BI->getAs<CFGStmt>())
-          O(const_cast<Stmt*>(stmt->getStmt()));
+          O(const_cast<Stmt *>(stmt->getStmt()));
       }
   }
 
